sycl/test/regression/device_num.cpp

```diff
@@ -44,7 +44,7 @@ int main() {
   deviceNum = std::atoi(envVal);
 
   auto devices = device::get_devices();
-  if (devices.size() >= deviceNum) {
+  if (devices.size() > deviceNum) {
```

Comment:
I think you should add running with SYCL_DEVICE_FILTER=0 too, ah?

Reply:
Yes, is is included. That's the first RUN at line 3.
So, in that case devices.size() > 0 will run the test.
Problem was that devices.size() == deviceNum means that deviceNum is over the number of devices available in the system.

Reply:
Right, I didn't scroll far enough. LGTM.